src/test/shell/bazel/verify_workspace.sh

```diff
@@ -46,14 +46,18 @@ function test_verify_urls() {
   # Find url-shaped lines, skipping jekyll-tree (which isn't a valid URL), and
   # skipping comments.
   invalid_urls=()
+  checked_urls=()
   for file in "${WORKSPACE_FILES[@]}"; do
     for url in $(grep -E '"https://|http://' "${file}" | \
       sed -e '/jekyll-tree/d' -e '/^#/d' -r -e  's#^.*"(https?://[^"]+)".*$#\1#g' | \
       sort -u); do
-      #echo "Checking ${url}"
-      if ! curl --head -silent --fail --output /dev/null --retry 3 "${url}"; then
-        #fail "URL ${url} is invalid."
-        invalid_urls+=("${url}")
+      # add only unique url to the array
```

Comment:
The sort -u should be keeping things unique. You should investigate why it is not and fix that instead.